scheduler-driver/src/test/java/gov/nasa/jpl/aerie/scheduler/PrioritySolverTest.java

```diff
@@ -119,8 +117,16 @@ private static PlanInMemory makePlanAB012(Problem problem) {
   }
 
   /** matches activities if they agree in everything except the (possibly auto-generated) names **/
-  private static final Correspondence<SchedulingActivityDirective, SchedulingActivityDirective> equalExceptInName = Correspondence.from(
-      SchedulingActivityDirective::equalsInProperties, "matches");
+  public static void assertActivitiesListsEquals(List<SchedulingActivityDirective> expected, List<SchedulingActivityDirective> actual) {
+    assertEquals(expected.size(), actual.size());
+
+    final var sortedExpected = expected.stream().sorted(Comparator.comparing(SchedulingActivityDirective::startOffset)).toList();
+    final var sortedActual = actual.stream().sorted(Comparator.comparing(SchedulingActivityDirective::startOffset)).toList();
```

Comment:
we may have a lot of activities starting at the same time

Reply:
I've updated the helper method to account for that. It's also been moved into `TestUtility`